pkg/microservice/aslan/core/common/service/registry/service.go

```diff
@@ -451,3 +457,102 @@ func (s *SwrService) GetImageInfo(option GetRepoImageDetailOption, log *zap.Suga
 
 	return &commonmodels.DeliveryImage{}, nil
 }
+
+type EcrService struct {
```

Comment:
You can consider abstracting an `interface`, returning the `interface`, and then hiding the implementation in the file through `unexported struct`.